### DISCUSSION TITLE
Litmus agent install error

### DISCUSSION THREAD

**Discussion by singhmd** [litmuschaos/litmus#3545] [Q&A] [answered]
Hello,
Getting below error while trying to add litmus agent in interactive mode in EKS. The default ChaosCenter account is admin which obviously don't have access to eks clusters. How/where to set a account with access? Thanks
 
Project list:
1.  admin's project

Select a project [Range: 1-1]: 1

Installation Modes:
1. Cluster
2. Namespace

Select Mode [Default: cluster] [Range: 1-2]: 1

🏃 Running prerequisites check....
🔑 clusterrole ❌
🔑 clusterrolebinding ❌
🚫 You don't have sufficient permissions.
🙄 Please use a service account with sufficient permissions.

**Comment by neelanjan00** [comment, marked as answer]
I guess for the installation of Litmus in Cluster scope, you need clusterrole and clusterrolebinding permissions. Otherwise, you can just use the Namespace scope, assuming you have permission for that. You may want to check with your sysadmin about this since they will have the complete permissions to set up LitmusChaos with Cluster scope.
